README.md

```diff
@@ -186,6 +186,29 @@ if err != nil {
 }
 ```
 
+### Handling Relationship Queries
+
+When querying Salesforce objects, it's common to access fields that are related through parent-child or lookup relationships. For instance, querying `Account.Name` with related `Contact` might look like this:
+
+##### Example SOQL Query
+```sql
+SELECT Id, Account.Name FROM Contact
+```
+
+#### Corresponding Go Structs
+To effectively handle the data returned by this query, define your Go structs as follows:
+
+```go
+type ContentDocumentLink struct {
+    Id       string
+    Account Account
+}
+
+type Account struct {
+    Name string
```

Comment:
Don't you have a tag thing to define?


```suggestion
    Name string `soql:"name"`
```


This could be useful for Salesforce formula fields that have strange name, it may be addressed with another PR if it doesn't exist yet

Reply:
@ccoVeille Only if you are using the `QueryStruct()` method, but that's not necessary for this example.